### PR TITLE
Display waterway brunnel

### PR DESCRIPTION
**Pull request by frodrigo**
#13

### CODE REVIEW

File: style.json [hunk 1]
```diff
@@ -230,15 +230,172 @@
...
+              1
+            ],
+            [
+              20,
+              2
+            ]
+          ]
+        },
+        "line-opacity": 1,
+        "line-offset": 1,
```
Comment (jirik) [inline style.json]: Why there are `line-offset` and `line-translate` used? What about using `line-gap-width` only?

File: style.json [hunk 2]
```diff
@@ -230,15 +230,172 @@
         ]
       }
     },
+    {
+      "id": "waterway-bridge-case",
```
Comment (jirik) [inline style.json]: Now waterway bridges are drawn under buildings, roads, etc.:
image

Please move two bridge layers (`waterway-bridge-case` and `waterway-bridge`) prior to other bridge layers, more precisely between `railway` and `bridge_minor case` layers.

File: style.json [hunk 3]
```diff
@@ -230,15 +230,172 @@
...
+          "brunnel",
+          "bridge"
+        ]
+      ],
+      "layout": {
+        "line-cap": "butt",
+        "line-join": "miter"
+      },
+      "paint": {
+        "line-color": "#dedede",
```
Comment (jirik) [inline style.json]: Please consider some darker color, e.g. `#bbb` (used in the image above). With `#dedede` the casing is not quite visible over residential landuse.

**Comment by jirik** [comment]
@frodrigo Thanks. I have made some changes to your PR (see three comments above). The biggest change is moving bridges close to other bridge layers to improve z-ordering.

Are you OK with these changes?

**Comment by frodrigo** [comment]
Look good to me.
About the bridge-case color, I just reuse the color of road bridge-case.

**Comment by jirik** [comment]
Thanks again for your work, merging.

**Pull request merged**